Entity/Media.php

```diff
@@ -373,17 +375,17 @@ public function getFilters()
     }
 
     /**
-     * Clear all filters
+     * Clear all filters.
```

Comment:
This dot on every comment is useless. We already disabled this on other projects